Ontology/Capability.json

```diff
@@ -30,6 +30,15 @@
       "name": "name",
       "schema": "string",
       "writable": true
+    },
+    {
+      "@type": "Property",
+      "name": "lastMeasuredTime",
+      "displayName": {
```

Comment:
displayName should be "last measured time".

Add:

"description": {
   "en": "Time when the host of the Capability captured the value of the sensor, setpoint, actuator"
}

Reply:
Ah good point, fixing that

Reply:
The "displayName" is still showing what should be the "description".